README.md

```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+# Subscriptions Customer Portal
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Description
 
-## Available Scripts
+This open-source project serves as a starting point for developers creating a customer portal from scratch that is integrated with Bold Subscriptions.
 
-In the project directory, you can run:
+## Installation
 
-### `npm start`
+Use the following steps to set up and configure this project:
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+1. Fork the repo into your workspace.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+2. Clone your copy of the repo using the following command:
 
-### `npm test`
+    ```
+    $ git clone https://github.com/path/to/your/forked/version
+    ```
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+3. Copy the `.env.example` file using the following command and update any values as necessary:
 
-### `npm run build`
+    ```
+    $ cp .env.example .env
+    ```
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+4. Ensure you are on the correct Node version using the following commands:
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+    ```
+    $ nvm install 16
+    $ nvm use 16
+    ```
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+5. Install, build, and run yarn using the following commands:
 
-### `npm run eject`
+    ```
+    $ yarn
+    $ yarn build
+    $ yarn start
```

Comment:
Since we have committed a `package-lock.json` and not a `yarn.lock` I would vote that we change this example to npm.

```
$ npm install
$ npm run build
$ npm run start
```